docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -141,44 +149,148 @@ Restrictions
 Relative Risks
 ^^^^^^^^^^^^^^
 
-The affected causes with the Vitamin A deficiency cause in GBD 2017 include
+The causes affected by the Vitamin A Deficiency risk in GBD 2017 include
 :ref:`lower respiratory infections <2017_cause_lower_respiratory_infections>`,
-:ref:`diarrhoeal diseases <2017_cause_diarrhea>`, :ref:`measles
+:ref:`diarrhoeal diseases <2017_cause_diarrhea>`, and :ref:`measles
 <2017_cause_measles>`. The relative risks for these causes appear in Table 4 on
-p. 112 of [GBD-2017-Risk-Appendix-VAD]_.
+p. 112 of [GBD-2017-Risk-Appendix-VAD]_, copied here for reference:
+
+.. _gbd_2017_vad_relative_risk_table:
 
 .. list-table:: Pooled relative risks for risk-outcome pairs included in GBD 2017
-	:widths: 15 13 15 15
-	:header-rows: 1
-
-	* - Cause
-	  - GBD 2016 RR
-	  - GBD 2017 RR
-	  - Include in GBD 2017
-	* - Diarrhea
-	  - 1.6 (1.21 - 2.02)
-	  - 2.35 (2.17 - 2.54)
-	  - Yes
-	* - Measles
-	  - 2.4 (1.61 - 3.48)
-	  - 2.76 (2.01 - 3.78)
-	  - Yes
-	* - Lower Respiratory Infections (LRI)
-	  - 
-	  - 1.23 (1.03 - 1.48)
-	  - Yes
-	* - Meningitis
-	  - 
-	  - 3.2 (0.69 - 14.75)
-	  - No (not significant)
-	* - Malaria
-	  - 
-	  - 3.65 (2.23 - 5.97)
-	  - No (only one study)
-	  
+  :widths: 15 13 15 15
+  :header-rows: 1
+
+  * - Cause
+    - GBD 2016 RR
+    - GBD 2017 RR
+    - Include in GBD 2017
+  * - Diarrhea
+    - 1.6 (1.21 - 2.02)
+    - **2.35 (2.17 - 2.54)**
+    - Yes
+  * - Measles
+    - 2.4 (1.61 - 3.48)
+    - **2.76 (2.01 - 3.78)**
+    - Yes
+  * - Lower Respiratory Infections (LRI)
+    -
+    - **1.23 (1.03 - 1.48)**
+    - Yes
+  * - Meningitis
+    -
+    - 3.2 (0.69 - 14.75)
+    - No (not significant)
+  * - Malaria
+    -
+    - 3.65 (2.23 - 5.97)
+    - No (only one study)
+
+The above relative risks for GBD 2017 should be interpreted as rate ratios for
+the incidence rates of diarrhea, measles, and LRI.
+
 Vivarium Modeling Strategy
 --------------------------
 
+We will use an **exposure model** (or **prevalence-only model** or **propensity
+model**) for a vitamin A deficiency, in which each simulant is initialized with a "propensity" for vitamin A deficiency, and the simulant's vitamin A status is determined by comparing this
```

Comment:
I don't know why we call this propensity except out of habit.  I think it might be clearer just to call it a percentile.

Reply:
For use with our standard sampling technique: https://en.wikipedia.org/wiki/Inverse_transform_sampling

Reply:
Ugh, it's not a percentile -- it's the inverse of the percentile/quantile under the quantile function, i.e. the probability of lying below the corresponding quantile.  I can't find a standard terminology for this. Perhaps "continuous [rank statistic](https://en.wikipedia.org/wiki/Ranking#Ranking_in_statistics)"?

See https://en.wikipedia.org/wiki/Quantile and https://en.wikipedia.org/wiki/Quantile_function

Reply:
Man, I've been thinking about percentile wrong for a while.  My stats professors would be ashamed.

Reply:
There really isn't a name for this.  Huh.

Reply:
Aha! Apparently it's called the [percentile rank](https://en.wikipedia.org/wiki/Percentile_rank) (or presumably quantile rank if we're talking about values in [0,1] instead of [0,100]).

So if Q is the quantile function and p is a probability, then q = Q(p) is a quantile with quantile rank p, i.e. a p-quantile. Or in reverse, if F is the CDF and q is an arbitrary value of the random variable, then the probability p = F(q) is the quantile rank of q, so q is a p-quantile. (I'm ignoring some subtleties about nonuniqueness when F is discontinuous or has constant pieces.)

I've been confused about this too.  But it's the statistics professors' fault!  For not laying out the definitions clearly.